### PR TITLE
Update high-density kubemark job

### PR DESCRIPTION
**Pull request by gmarek**
I already bumped quota for the project. @fejta - PTAL, as I'm not familiar with this new framework, so I don't know if I shouldn't change something else (e.g. timeout).

cc @wojtek-t

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1084)

### CODE REVIEW

**Comment by gmarek** [comment]
It's a non blocking build, so I'm merging this change. This way we'll have some data if it fails.

**Comment by fejta** [comment]
(sorry thought it already sent my comments)

**Comment by fejta** [comment]
(and thanks for merging!)

File: jobs/ci-kubernetes-kubemark-high-density-100-gce.sh [hunk 1]
```diff
@@ -40,12 +40,12 @@ export KUBEMARK_TESTS="\[Feature:HighDensityPerformance\]"
...
 export MASTER_SIZE="n1-standard-2"
-export NODE_SIZE="n1-standard-4"
+export NODE_SIZE="n1-standard-8"
 export KUBE_GCE_ZONE="us-east1-d"
 export KUBEMARK_MASTER_SIZE="n1-standard-4"
-export KUBEMARK_NUM_NODES="100"
+export KUBEMARK_NUM_NODES="600"
 # The kubemark scripts build a Docker image
 export JENKINS_ENABLE_DOCKER_IN_DOCKER="y"
 export KUBE_NODE_OS_DISTRIBUTION="gci"
```
Comment (gmarek) [inline jobs/ci-kubernetes-kubemark-high-density-100-gce.sh]: It would be great if it can be made clearer, like having a variable for it or sth.

**Pull request merged**